Hashing/README.md

```diff
@@ -0,0 +1,64 @@
+## Introduction to Hashing:
+

```

Comment:
Adding a table of contents section shall improve the reader's accessibility to various areas of the micro bytes.